app/src/Screens/RegisterProject.svelte

```diff
@@ -58,8 +58,8 @@
       response = await mutate(client, {
         mutation: REGISTER_PROJECT,
         variables: {
-          project_name: project_name,
-          org_id: org_id
+          projectName: projectName,
+          orgId: "rad123"
```

Comment:
This has to be replaced, once the org creation is implemented. It might make sense to indicate this already now and keep the `org_id` variable outside of this function, but hardcode it for now.

Reply:
To avoid possible confusion, I'd suggest renaming "rad123" to "monadic" to 1) be consistent with the other usages and 2) to not confuse rad(icle) as a project.